databus-client-jersey2/src/main/java/com/bazaarvoice/emodb/databus/client/discovery/PartitionedDiscovery.java

```diff
@@ -57,7 +57,7 @@ protected URI getBaseUriFromDiscovery() {
     private List<Integer> computeHashCodes(String id) {
         List<Integer> list = Lists.newArrayListWithCapacity(100);
         for (int i = 0; list.size() < 100; i++) {
-            Hasher hasher = Hashing.md5().newHasher();
+            Hasher hasher = Hashing.sha512().newHasher();
```

Comment:
needs to be reverted (and the check can suppressed)

Reply:
the changes were reverted